### PR TITLE
Added steps to send and receive a payload in getting started

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added steps to send and receive a payload in getting started

### CODE REVIEW

**Comment by Krish1979** [comment]
Quorum talks to Tessera via IPC as default, so I wouldn't change ipc connection on Q2T server to http port

**Comment by NicolasMassart** [comment]
`/receive` method will be changed from GET to POST soon in Tessera. Is this PR able to wait for the change? I made it a draft for now until we decide.

**Comment by MadelineMurray** [comment]
> Quorum talks to Tessera via IPC, so I wouldn't change ipc connection on Q2T server to http port

@Krish1979 - will do. Is there a specific reason for supporting http on the Q2T server if it's not used?

**Comment by Krish1979** [comment]
Q2T connection is http over IPC and not over web port as default but we could include support over http port as well. Also to keep it generic we will be connecting Besu to Tesser over Q2T server on a http port. I would wait till we add Besu <-> Tessera operability and then update the document to reflect all the updates.

File: docs/HowTo/Get-started/Start-Tessera.md [hunk 1]
```diff
@@ -111,3 +111,88 @@ Use the `upcheck` method to confirm Tessera is up and running.
...
+    ```bash
+    {
+      "payload": "SGVsbG8sIFdvcmxkIQ==",
+      "affectedContractTransactions": [],
+      "execHash": "",
+      "privacyFlag": 0
+    }
+    ```
+
+Where `SGVsbG8sIFdvcmxkIQ==` is `Hello, World!` in Base64.
```
Comment (NicolasMassart) [inline docs/HowTo/Get-started/Start-Tessera.md:198]:
```suggestion
    Where `SGVsbG8sIFdvcmxkIQ==` is `Hello, World!` in Base64.
```
Should be part of the result tab, so need to add spaces to increment it.

File: docs/HowTo/Get-started/Start-Tessera.md [hunk 2]
```diff
@@ -111,3 +111,88 @@ Use the `upcheck` method to confirm Tessera is up and running.
...
+        "privacyFlag": 0,
+        "affectedContractTransactions": [],
+        "execHash": ""
+      }'
+    ```
+
+=== "Result"
+
+    ```bash
+    {"key": "5jZPbwuxncr7qrL9esCTnz3PByLlD50kdZy5JkW/JMKcFeTl7I3NeLIsje4C410EYb93HabhddoAVGpwgYF1LQ=="}
```
Comment (NicolasMassart) [inline docs/HowTo/Get-started/Start-Tessera.md:155]: can you explain what this key is? Public, private? What care do we have to take with this?

**Review by NicolasMassart — APPROVED** [review]
tested and approved.
Just fix the spacing indicated on the comment.

**Comment by NicolasMassart** [comment]
Converted this PR to draft as it probably requires work to be up to date since Nov 2020...

**Comment by macfarla** [comment]
stale

**Pull request closed without merging**